src/main/steps/c100-rebuild/c100sequence.ts

```diff
@@ -515,4 +516,9 @@ export const C100Sequence: Step[] = [
     showInSection: Sections.C100,
     getNextStep: () => C100_C1A_SAFETY_CONCERNS_OTHER,
   },
+  {
+    url: C100_GET_CASE,
+    showInSection: Sections.C100,
+    getNextStep: () => C100_MIAM_MEDIATOR_CONFIRMAION,
```

Comment:
Let's add nextStep same as url - C100_GET_CASE else it might confuse why we are navigating MIAM though we are not dependent on this.

Reply:
Made this change